README.md

```diff
@@ -41,6 +41,16 @@ $orcidIds = OrcidId::extract('0000-0002-0088-0058');
 
 Return an array of ORCID identifiers from a given string. Will return an empty array if no matches are found.
 
+### `RepecId::extract($str)`
+
+```php
+use Altmetric\Identifiers\RepecId;
+
+$repecIds = RepecId::extract("RePEc:wbk:wbpubs:2266\nRePEc:inn:wpaper:2016-03");
```

Comment:
This affects the other examples but I wonder if it'd be helpful to also include the expected return values here to show both input _and_ output?